python_modules/dagster-graphql/dagster_graphql/implementation/pipeline_run_storage.py

```diff
@@ -1,27 +1,83 @@
+from enum import Enum
+from dagster import check
+from threading import Event, Thread
+from time import sleep
+
+import gevent
+
+
+class State(Enum):
+    # initialized
+    NULL = 0
+    # streaming the existing event log entries
+    LOADING = 1
+    # watching for new writes
+    WATCHING = 2
+
+
+CHUNK_SIZE = 10000
+
+
 class PipelineRunObservableSubscribe:
```

Comment:
do we need to do something similar for the other subscription calls (compute logs)?

Reply:
I don't think so, at least not urgently